docker-compose.dev.yml

```
@@ -0,0 +1,43 @@
+        reservations:
+          memory: 2G
+    environment:
+      - DOCKER_BUILDKIT=1
+      - DOCKER_CLI_EXPERIMENTAL=enabled
+      # Elixir/Mix environment variables to reduce memory usage
+      - MIX_ENV=prod
+      - ERL_AFLAGS="-kernel shell_history enabled"
+    command: >
+      bash -c "
+        # Install required packages
+        apt-get update && 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)